draft-ietf-trans-rfc6962-bis.md

```diff
@@ -1236,6 +1236,11 @@ If the certificate is logged, an SCT MUST be issued. Logging the certificate is
 useful, because monitors ({{monitor}}) can then detect these encoding errors,
 which may be accepted by some TLS clients.
 
+If `submission` is an accepted trust anchor whose certifier is neither an
+accepted trust anchor nor the first element of `chain`, then the log MUST return
```

Comment:
Is it clear, though, that a submission of an accepted trust anchor shouldn't be accepted unless there's *another* accepted trust anchor in the chain? Could anyone mis-interpret this section to mean that a non-self-signed accepted trust anchor may be submitted with its chain and it would still be accepted because it contains a trust anchor?